sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationChecker.scala

```diff
@@ -105,7 +105,7 @@ object UnsupportedOperationChecker extends Logging {
     case d: Deduplicate if d.isStreaming && d.keys.exists(hasEventTimeCol) => true
     case d: DeduplicateWithinWatermark if d.isStreaming => true
     case t: TransformWithState if t.isStreaming => true
-    case t: TransformWithStateInPandas if t.isStreaming => true
```

Comment:
Don't we need to retain this also ?

Reply:
The class name of physical node has changed, so no need to do that.